supabase/functions/_backend/private/plugins/stats.ts

```diff
@@ -69,6 +69,23 @@ export const jsonRequestSchema = z.object({
   message: INVALID_STRING_DEVICE_ID,
 })
 
+function requestData(c: Context, app_id: string, version_name: string) {
+  const appOwnerRaw = supabaseAdmin(c)
```

Comment:
There are many calls to stats made by plugin not using Capgo cloud, so this will just create 2 requests instead of one in that case.
Not sure it's better approach

Reply:
I don't think there is a case when you send only 1 request - i checked the old code